### PR TITLE
feat(docs): update Getting Started instructions to write home directory files via the worker config

### PR DESCRIPTION
**Pull request by meln1k**
# Description

# Docs Version(s)

- [x] `edge`
- [x] `3.X`
- [ ] `2.X`

# To-Dos

- [x] Format with `yarn format`
- [x] Lint with `yarn lint`
- [x] Build successfully with `yarn build`

# Issues Fixed

- Fixes #XXXX

# Code of Conduct

By submitting this pull request, I agree to follow the [code of conduct](https://resoto.com/code-of-conduct).

### CODE REVIEW

File: docs/getting-started/configure-resoto/aws.md [hunk 1]
```diff
@@ -59,6 +59,37 @@ https://youtu.be/6_nxUM0iFx4
 3. Make your `credentials` file available to Resoto at `/home/resoto/.aws`:
 
    <Tabs groupId="install-method">
+   <TabItem value="worker-config" label="Resoto Config">
```
Comment (aquamatthias) [inline docs/getting-started/configure-resoto/aws.md:62]: Can we add a note on the Kubernetes tab, that informs the user about this option?

**Review by TheCatLady — CHANGES_REQUESTED** [review]
So, a couple notes:

1. `worker-config` is not an install method, and this breaks tab logic on other pages if we allow this value to be set here.
2. I think we should be opinionated about how users add this config. If the Resoto Worker config is the better, more straightforward way to add credential data, we should replace the existing instructions rather than add another option to confuse users.

Thoughts?

**Comment by aquamatthias** [comment]
> 1. `worker-config` is not an install method, and this breaks tab logic on other pages if we allow this value to be set here.

I see your point. So we should document this as part of the installation method.
Going forward, I would opt for this:
- let's document in `resoto.worker` configuration how to mount files 
- in our installation section we describe how to provide the files via configuration and refer to this documentation if people want to provide files directly

wdyt?

**Comment by TheCatLady** [comment]
> > 1. `worker-config` is not an install method, and this breaks tab logic on other pages if we allow this value to be set here.
> 
> I see your point. So we should document this as part of the installation method. Going forward, I would opt for this:
> 
> * let's document in `resoto.worker` configuration how to mount files
> * in our installation section we describe how to provide the files via configuration and refer to this documentation if people want to provide files directly
> 
> wdyt?

I think that sounds good! I think that the fewer choices we require users to make in the onboarding process, the better -- we should make it as easy and simple as possible to get Resoto up and running, especially since users might not know how to make those decisions at the point where they first read the "Getting Started" section of our docs.

I think being opinionated about adding the config via the Worker config will also have other benefits -- it's easier to write documentation that applies regardless of install method, and users would be able to make these changes without updating volume mounts, etc. outside of the application. Install-specific instructions should always be the exception rather than the default, if possible.

**Comment by aquamatthias** [comment]
We should not merge this PR, until we release the current main.

**Review by TheCatLady — APPROVED** [review]

**Pull request merged**